File: brainteaser.js

let fox = 'the quick brown fox'

const brainteaser = () => {
  let splitWords = fox.split(' ')
  let splitLetters = splitWords.map((word) => {
    let capFirstLetter = word.split('').shift().toUpperCase()
    let otherLetters = word.split('')
    otherLetters.shift()
    otherLetters.unshift(capFirstLetter)
    return otherLetters.join('')
  })
  return splitLetters.join(' ')
}

console.log(fox)
console.log(brainteaser())

File: brainteaser.rb

@fox = "the quick brown fox"


def cap_this
  cap_fox = @fox.split(' ').map(&:capitalize).join(' ')
  cap_fox
end


def brainteaser
  cap_fox = @fox.split(' ')
  new_arr = cap_fox.map do |word|
    first_letter = word.split('').shift.swapcase
    letters = word.downcase.split('').drop(1)
    letters.unshift(first_letter).join('')
  end
  new_arr.join(' ')
end

p @fox
p brainteaser

# p @fox.swapcase

